### ISSUE TITLE
Hydro thunder broke with recent commits

### ISSUE THREAD

**Issue by fzurita** [mupen64plus/mupen64plus-core#391]
Hydro thunder doesn't seem to be booting up any more. Something recent must had broken it. I tried interpreter mode, the CXD4 RSP, and Angrylion. 

Different CountPerOp settings are not helping either.

**Comment by loganmc10** [comment]
I can't reproduce this on Linux. I tested dynarec, cached and pure interpreter. I also tested HLE and LLE (using GLideN64 and Angrylion).

I was surprised to read this, since this is a game I tested quite a bit with the changes to the RSP interrupt timing. I wonder what the difference could be.

**Comment by Gillou68310** [comment]
I'm not able to reproduce it on windows either.
@fzurita could you share your rom MD5?
I tested with the US rom MD5: 54F43E6B68782E98CAABEA5E7976B2BE

**Comment by Gillou68310** [comment]
I also tested on my odroid armhf and no issue

**Comment by fzurita** [comment]
Ok, it's possible I have a bad dump. My md5 does not match yours.

**Comment by loganmc10** [comment]
It's my fault. I realized the other day that I've been missing a step when I add stuff to rom.c. Hydro Thunder needed AudioSignal=1 in mupen64plus.ini, but since your MD5 doesn't match, and I did it wrong in rom.c, it's not copying AudioSignal=1 over from the main INI entry for Hydro Thunder.

Anyway just recently the PR was merged that got rid of the need for AudioSignal, so if you update the core it'll probably work.  I also corrected the only INI entry that still needs correcting in this commit: https://github.com/mupen64plus/mupen64plus-core/pull/388/commits/002bb14892d7a6ff62a1d2ec93bedbb57130189e as part of PR #388
